components/brave_wallet_ui/panel/container.tsx

```diff
@@ -226,16 +217,9 @@ function Container () {
       setReadyToConnect(false)
     }
   }
-  const unlockWallet = () => {
-    dispatch(WalletActions.unlockWallet({ password: inputValue }))
-    setInputValue('')
-  }
 
-  const handlePasswordChanged = (value: string) => {
-    setInputValue(value)
-    if (hasIncorrectPassword) {
-      dispatch(WalletActions.hasIncorrectPassword(false))
-    }
+  const unlockWallet = (password: string) => {
+    dispatch(WalletActions.unlockWallet({ password }))
```

Comment:
Input clearing no longer needed since the lock-screen component holds its own state now